diff --git a/dist/database/index.js b/dist/database/index.js
index aa5b1fd17c6682edc26863c12b09c6b7d70e465a..ee95908cb4e774388f008b5a2d647d0e5cee04a1 100644
--- a/dist/database/index.js
+++ b/dist/database/index.js
@@ -8,15 +8,9 @@ async function createDb(collectionName, options = {}) {
 	dbPromise ??= initDb(options);
 	let raws;
 	let _token;
-	if (typeof window === "undefined") {
-		const { config } = await import("../init.js");
-		raws = config.compresseds;
-		_token = config.token;
-	} else {
-		const _raws = await import("virtual:v-content/compressed");
-		raws = _raws.default.compresseds;
-		_token = _raws.default.token;
-	}
+	const _raws = await import("virtual:v-content/compressed");
+	raws = _raws.default.compresseds;
+	_token = _raws.default.token;
 	if (_token !== token) {
 		token = _token;
 		seeded.clear();
@@ -34,6 +28,6 @@ async function initDb({ path = ".content.db", filename = "content.db" }) {
 	} else {
 		const { createServerDb } = await import("./server.js");
-		db = createServerDb(path);
+		db = createServerDb(process.env.LAMBDA_TASK_ROOT ? "/tmp/content.db" : path);
 	}
 	return db;
 }
